docs/subnets/cross-subnet-communication.md

```diff
@@ -0,0 +1,46 @@
+# Cross-Subnet Communication
+
+Avalanche Warp Messaging (AWM) enables native cross-Subnet communication and allows 
+[Virtual Machine (VM)](./README.md#virtual-machines) developers to utilize
+any-to-any communication between the validators of two Subnets. The content of the messages can contain arbitrary payloads. 
+
+## Use Cases 
+
+Use cases for AWM may include but is not limited to:
+
+- Orcale Networks: Connecting a Subnet to an oracle network is a costly process. AWM makes it easy for oracle networks to broadcast their data from their origin chain to other Subnets.
+- Token transfers between Subnets 
+- State Sharding between multiple Subnets
+
+## Elements of Cross-Subnet Communication
+
+The communication consists of the following four steps:
+
+![image showing four steps of cross-Subnet communication: Signing, aggregation, Delivery and Verification](/img/cross-Subnet-communication.png)
```

Comment:
The image name is wrong because of the capital letter. The name should be cross-subnet-communication.png